### ISSUE TITLE
[Error] IA 3.0.3-beta4 is not working and Found error between Mythicmob and

### ISSUE THREAD

**Issue by ArtifactyNight** [PluginBugs/Issues-ItemsAdder#1555]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already asked on the **#💬ia-community-help** channel on **Discord** to know if anyone already has a solution for the issue.

### Discord tag (optional)

NightKunGz#0790

### What happened?

error  when i update the plugin and plugin is disabled

### Steps to reproduce the issue

1.) install MythicMob (premium stable version)
2.) install IA 3.0.3-beta4

### Server version

```
This server is running Paper version git-Paper-216 (MC: 1.18.1) 
(Implementing API version 1.18.1-R0.1-SNAPSHOT) (Git: 30cb7d0)
```

### ItemsAdder Version

3.0.3-beta4

### ProtocolLib Version

4.8.0-SNAPSHOT-b538

### LoneLibs Version

1.0.18

### FULL server log

https://mclo.gs/ThQ1P3x

### Error (optional)

```shell
https://mclo.gs/lpiV3b9
```

### ItemsAdder config.yml

```shell
metrics: true
update-checker: false
disable-plugin-loading-messages_DONT_ASK_ME_FOR_SUPPORT_IF_YOU_SET_TRUE__THANKS: false
resource-pack:
  hosting:
    no-host:
      enabled: false
    auto-external-host:
      enabled: false
    self-host:
      enabled: true
      server-ip: 127.0.0.1
      pack-port: 10024
    external-host:
      enabled: false
      url: https://download.mc-packs.net/pack/8b231e7d902d96d67c9d1bf09867aa21a58781de.zip
      skip-url-file-type-check___DONT_ASK_HELP_IF_SET_TRUE: false
  apply-on-join: true
  kick-player-on-decline: false
  kick-player-on-fail: false
  delay-ticks: 1
  notify-other-plugins-interfering: true
  title:
    enabled: true
  protect-player:
    black-screen: true
    hide-hud: true
    cancel-movement: true
    lock-player: true
  command:
    usage-cooldown-seconds: 0
  custom-font:
    enabled: false
    name: ipaexg.ttf
    shift: '[0,0]'
    size: 11.0
    oversample: 2.0
  thin-font:
    enabled: false
  zip:
    protect-file-from-unzip:
      enabled: true
      extreme: true
    overwrite-vanilla-models: false
    overwrite-vanilla-lang-json: false
    fix_transparent_noteblocks__disable-for-more-performance: false
    ignore-files-ext:
    - db
    - yml
    - xcf
    - html
    compress-png-files: true
    compress-json-files: true
    generate-custom-armors-textures:
      vanilla_1_17: true
      optifine: false
    disable-overwrite-leather-armor-layers-textures: false
  items:
    CustomModelData-starting-value:
      ALL: 1
      LEATHER_HORSE_ARMOR: 20000
      DIAMOND: 11000
# DIAMOND: 11000
# LEATHER_HELMET: 1
# LEATHER_CHESTPLATE: 1
# LEATHER_LEGGINGS: 1
# LEATHER_BOOTS: 1
# PAPER: 1
config_files:
  lang: en
  dictionaries-lang: en
  extract-default-items: false
  extract-default-resources: false
blocks:
  light-api: true
  fix-glitched-blocks:
    enabled: true
    only-new-chunks: false
    search-also-in-these-biomes:
      enabled: false
      biomes:
      - BEACH
      - BIRCH_FOREST
      - BIRCH_FOREST_HILLS
      - FOREST
      - MOUNTAINS
      - MOUNTAIN_EDGE
      - GRAVELLY_MOUNTAINS
      - PLAINS
      - RIVER
  cancel-break-wrong-tool:
    enabled: true
    send-actionbar: true
  campfire:
    unlit-on-place: true
    allow-any-item: true
  disable-REAL: false
  disable-REAL_TRANSPARENT: false
  disable-REAL_NOTE: false
  disable-REAL_WIRE: false
  disable-FIRE: true
  fix-liquids-flow: true
  fix-floating-blocks: true
items:
  fix-show-attack-and-damage-as-lore: true
  give_death_location_maps: false
  hide-custom-durability-lore: true
liquids:
  enabled: false
huds:
  enabled: false
mobs:
  disable-MOB-behaviour__DISABLE_AT_YOUR_OWN_RISK: false
font_images:
  chat:
    enabled: true
    doublecheck-permission__disable-for-more-performance: false
    add-hover-suggestion: true
  command:
    enabled: true
    commandblocks: true
    excluded:
    - home
    - sethome
    - tpa
  sign:
    enabled: false
  book:
    enabled: false
  anvil:
    enabled: false
  inventory-title:
    enabled: true
  tab-header-footer:
    enabled: false
  vault-prefix-suffix:
    enabled: true
    update-every-ticks: 1200
  scoreboard-teams:
    enabled: false
    update-every-ticks: 3600
  player-display-name:
    enabled: true
    update-every-ticks: 1200
  replace-in-customitems-name-and-lore: true
effects:
  hide-scoreboard-numbers: true
  text-effects:
    enabled: true
    replace-in-customitems-name-and-lore: true
    chat:
      enabled: true
    sign:
      enabled: true
    book:
      enabled: true
    anvil:
      enabled: true
    tab-header-footer:
      enabled: false
entities:
  max-furniture-vehicles-per-chunk: 30
  allow-removing-old-furnitures_2_3_11: false
  custom-entitites:
    enabled: true
    interval-rendering-packets-ticks: 1
recipes:
  hide-vanilla-unrecognized-recipe-errors: true
  custom-recipes:
    unlock:
      all-on-join: false
      on-pickup:
        any: false
        ingredients: true
  remove-vanilla-recipes:
    enabled: true
    list:
      DIAMOND_SWORD: false
      IRON_INGOT: false
  fix-packet-too-large-kick: false
debug:
  items:
    log-files-loading: false
    log-items-loading: false
    log-items-loading-disabled: false
    log-font-images-loading: false
    log-huds-loading: false
    log-attribute-modifiers-loading: false
    log-warnings: false
    log-loots-loading: false
    log-loots-loading-disabled: false
    log-armors-loading: false
  recipes:
    log-recipes-loading: false
    log-recipes-loading-disabled: false
    log-recipes-removed-vanilla: false
    log-fix-packet-too-large: false
  resource-pack:
    log-extract-default-resources: false
    log-not-extract-default-resources-already-exist: false
  blocks:
    log-save: false
    log-save-on-unload: false
    log-mined-unknown-blocks: true
  worlds-populators:
    log-loading: false
    log-loading-disabled: false
    log-loading-no-populators: false
    log-loading-no-world: true
    log-generation: false
  trees-populators:
    log-loading: false
    log-loading-disabled: false
    log-loading-no-populators: false
    log-loading-no-world: true
    log-generation: false
  surface-decorators:
    log-loading: false
    log-loading-disabled: false
    log-loading-no-decorators: false
    log-loading-no-world: true
    log-generation: false
  player-stats:
    log-save: false
  performance:
    loading-ms: true
  huds:
    log-triggers: false
  other:
    log-dictionaries-loading: false
    log-liquids-loading: false
    log-minecraft_lang_overwrite-loading: false
  categories:
    log-total-loaded: true
    log-single-loaded: false
    log-single-loaded-overwrite: false
    log-single-skip: false
  entities:
    log-files-loading: false
    log-entities-loading: false
    log-entities-loading-disabled: false
loots:
  allow-loots-drop-from-spawners-using-silk-touch: false
```

**Comment by MrJorman** [comment]
+++

**Comment by LoneDev6** [comment]
Update Mythicmobs

**Comment by ArtifactyNight** [comment]
I'm using Mythicmob Premium version
so i need to move it to dev build version?

![image](https://user-images.githubusercontent.com/53427253/158535560-b6e2d505-97e9-4f49-b393-e1f4fc3533a1.png)

**Comment by LoneDev6** [comment]
To me this seems to be a messy way of organizing versions.
I expected the latest free version to be less recent than the premium one, but it's the opposite? what?

I just based the update on the one available on Spigot: 5.0.1.
https://www.spigotmc.org/resources/5702/
